appveyor.yml

```diff
@@ -3,6 +3,11 @@
 # file, make changes to conda-forge.yaml and/or recipe/meta.yaml, and run
 # "conda-smithy regenerate".
 
+# Skipping commits with particular message or from user
+skip_commits:
+  message: .*
+  author: .*
```

Comment:
Yeah, it's not super elegant. Though if we can't get the ideal case, this is still an improvement.